### PR TITLE
add fcl and libccd

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/fcl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+  host:
+    - boost
```
Comment (scopatz) [inline recipes/fcl/meta.yaml]: Shouldn't this be `boost-cpp` or does it need BoostPython?

File: recipes/fcl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - flann
+    - libode
+    - libccd
+    - eigen
+    - octomap
+  run:
+    - boost
```
Comment (scopatz) [inline recipes/fcl/meta.yaml]: Same

**Comment by wolfv** [comment]
Thanks @scopatz for pointing the boost problem out.

**Comment by scopatz** [comment]
Thanks @wolfv!

**Pull request merged**